developer-rchain-coop/src/views/token-swap.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="masthead">
     <div id="logoWrapper">
-        <img src="/assets/rchain-redwhite-logo.png" alt="rchain">
+        <img src="https://developer.rchain.coop/assets/rchain-redwhite-logo.png" alt="rchain">
```

Comment:
this change looks spurious; was it on purpose?

Reply:
Leaving it in? No. I copied header info and changed it all to absolute links to test how my changes would look. Thanks.

Reply:
change reverted